pre-receive

```diff
@@ -42,6 +42,14 @@ case $(puppet --version) in
 esac
 
 while read -r oldrev newrev refname; do
+
+    # workaround: deleting a branch should skip the checks
+    if [[ $newrev == "0000000000000000000000000000000000000000" ]]; then
+        echo "Skipping file checks this is a branch delete..."
+        rm -rf "$tmptree"
```

Comment:
@brianlong2  It would be much safer to use `rmdir` instead of `rm`. With this we avoid problems mentioned by David in comment.